src/util/datetime.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <QDateTime>
+#include <QVariant>
+
+#include "util/assert.h"
+
+namespace mixxx {
+
+/// Common utility functions for safely converting and consistently
+/// displaying date time values.
+
+/// Obtain the local date time from an UTC date time.
+inline QDateTime localDateTimeFromUtc(
+        const QDateTime& dt) {
+    DEBUG_ASSERT(dt.timeSpec() == Qt::UTC);
```

Comment:
The original code called setTimeSpec(Qt::UTC) without any explanation why. I added this debug assertion to check that it is indeed unnecessary. 

Wouldn't do any harm other than displaying the time with an offset so I guess this is acceptable.

Reply:
The debug assertion was triggered so I reverted back to the original code.